lib/html/pipeline/image_filter.rb

```diff
@@ -0,0 +1,17 @@
+module HTML
+  class Pipeline
+    # HTML Filter that converts image's url into <img> tag.
+    # For example, it will convert
+    #   http://example.com/test.jpg
+    # into
+    #   <img src="http://example.com/test.jpg" alt=""/>.
+
+    class ImageFilter < TextFilter
```

Comment:
Cause I want to use the `@text` variable to get the raw text content, and I want to make sure the content uses this filter is always string.